docs/sop.rst

```diff
@@ -0,0 +1,135 @@
+Operating Clowder
+=================
+
+Two primary aspects of operating Clowder: Operating the apps managed by Clowder and operating
+Clowder itself.
+
+Contrary to many other deployment models where apps configure themselves, an application that is
+managed by Clowder will have configuration presented to it and is expected to use it. Clowder
```

Comment:
apps are "expected" to use it because there is no other way to obtain this information.  I know this is an SOP and not marketing material, but I would like to use this paragraph to emphasize the core value of Clowder by explicitly stating how we are removing the configuration effort from devs.